wrappers/python/pysetup.cmake.in

```diff
@@ -1,8 +1,13 @@
 set(ENV{OPENMM_INCLUDE_PATH} "@WRAPPER_INCLUDE_DIRS@")
+set(ENV{CC} "@CMAKE_C_COMPILER@")
+set(ENV{CXX} "@CMAKE_CXX_COMPILER@")
 file(TO_NATIVE_PATH "@CMAKE_INSTALL_PREFIX@/lib" OPENMM_LIB_PATH)
 set(ENV{OPENMM_LIB_PATH} "${OPENMM_LIB_PATH}")
 message("OPENMM_LIB_PATH = " $ENV{OPENMM_LIB_PATH})
 message("OPENMM_INCLUDE_PATH = " $ENV{OPENMM_INCLUDE_PATH})
+message("CC = " $ENV{CC})
+message("CXX = " $ENV{CXX})
```

Comment:
The point is that suppose I want to use the Intel compiler to build OpenMM.  Whatever Python is installed on the system was almost certainly built with GCC.  So the standard CFLAGS that distutils uses may not work for icpc or icc (since they correspond to gcc/g++).  So you might get errors about unrecognized flags.  I'll try this on my desktop when I get a chance to see if it really is a problem.

Since the SWIG wrappers are basically just boilerplate without any performance implications, there's not really any advantage to trying to force the Python wrappers to be built with icc/icpc.